keps/sig-auth/541-external-credential-providers/README.md

```diff
@@ -36,14 +36,14 @@
 
```

Comment:
Update the `kep.yaml` to indicate:

1. `status` `=` `implemented`
2. `stage` `=` `stable`
3. `latest-milestone` `=` `"v1.22"`
4. Enumerate the `metrics` we have

Reply:
e2d3cf87